public/docs/_examples/structural-directives/dart/lib/structural-directives.component.html

```diff
@@ -0,0 +1,109 @@
+<!-- #docplaster -->
```

Comment:
I think this filename should probably be `structural_directives_component.html`, to match the Dart file (not the TS equivalent).